pallets/subtensor/src/lib.rs

```diff
@@ -294,8 +298,11 @@ pub mod pallet {
     #[pallet::storage] // --- MAP ( hot ) --> cold | Returns the controlling coldkey for a hotkey.
     pub type Owner<T: Config> =
         StorageMap<_, Blake2_128Concat, T::AccountId, T::AccountId, ValueQuery, DefaultAccount<T>>;
-    #[pallet::storage] // --- ITEM ( delegate_limit ) --> Maximmu number of nominators per subnet validator
-    pub type DelegateLimit<T> = StorageValue<_, u32, ValueQuery, DefaultDelegateLimit<T>>;
+    #[pallet::storage] // --- ITEM ( delegate_limit ) --> Maximum number of nominators per hotkey
+    pub type NominatorLimit<T> = StorageValue<_, u32, ValueQuery, DefaultNominatorLimit<T>>;
```

Comment:
I think not, because we are counting unique nominators here, and one nominator can make multiple nominations that would only increase the counter by 1.